test/Integration/IntegrationTestBase.cs

```diff
@@ -60,91 +54,36 @@ public class IntegrationTestBase : IDisposable
         protected ITestOutputHelper TestOutput { get; private set; }
 
         /// <summary>
-        /// The port the Functions Host is running on. Default is 7071.
+        /// The port the Functions Host is running on. Start at 7081 since the function hosts started in
+        /// IntegrationTestFixture are already running on ports 7071 - 7080.
         /// </summary>
-        protected int Port { get; private set; } = 7071;
+        protected int Port { get; private set; } = 7081;
```

Comment:
I'd suggest defining a const for this in TestUtils next to the port helper method so that if we end up adding more ports later on we remember to update this.